src/tools/compiletest/src/runtest.rs

```diff
@@ -347,7 +348,7 @@ impl<'test> TestCx<'test> {
     fn run_rfail_test(&self) {
         let pm = self.pass_mode();
         let should_run = self.run_if_enabled();
-        let proc_res = self.compile_test(should_run, self.should_emit_metadata(pm));
+        let proc_res = self.compile_test(should_run, self.should_emit_metadata(pm), Vec::new());
```

Comment:
Why changing `compile_test` function at all, if in all places except one, last parameter will be `Vec::new()`? You can just add something like `compile_test_with_passes` and use it exactly in that one place.

Reply:
good point, I'll make a follow up with that change